src/ledger.js

```diff
@@ -911,11 +968,11 @@ export class LedgerSignMultisigTransaction extends LedgerBitcoinInteraction {
    * Input signatures produced will always have a trailing `...01`
    * {@link https://bitcoin.org/en/glossary/sighash-all SIGHASH_ALL}
    * byte.
-   * 
+   *
    * @returns {string[]} array of input signatures, one per input
    */
   async run() {
```

Comment:
Same.

Reply:
this one needs bc await call to signP2SHTransaction